src/GitVersionHelper.php

```diff
@@ -40,7 +40,13 @@ public static function getVersion()
         $fail = false;
         if (class_exists('\Symfony\Component\Process\Process')) {
             try {
-                $process = Process::fromShellCommandline($command, $path);
+                
```

Comment:
Just remove this new empty line and squash your commits, please.

Reply:
Is it okay now?